### PR TITLE
Refactor the 'length' and 'childElementCount' properties of the 'Code instumentaion' subsystem (#1379)

### PR DESCRIPTION
**Pull request by LavrovArtem**
part of #1379

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: test/client/fixtures/sandbox/code-instrumentation/setters-test.js [hunk 1]
```diff
@@ -298,8 +298,9 @@ test('body.innerHTML in iframe', function () {
     return createTestIframe({ src: getSameDomainPageUrl('../../../data/code-instrumentation/iframe.html') })
         .then(function (iframe) {
             var hasShadowUIRoot = function () {
-                var iframeBody = iframe.contentDocument.body;
-                var root       = iframeBody.children[iframeBody.children.length - 1];
+                var iframeBody             = iframe.contentDocument.body;
+                var bodyChildrenRealLength = nativeMethods.htmlCollectionLengthGetter.call(iframeBody.children);
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/code-instrumentation/setters-test.js]: `var bodyChildrenOriginLength = ...` there and below

File: src/client/utils/overriding.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import nativeMethods from '../sandbox/native-methods';
+
```
Comment (miherlosev) [inline src/client/utils/overriding.js]: Rename file to override-descriptor.js

Comment (miherlosev) [reply src/client/utils/overriding.js]: And export function as default

**Comment by LavrovArtem** [comment]
FPR

**Pull request merged**